torchbenchmark/models/llama/__init__.py

```diff
@@ -0,0 +1,50 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# This software may be used and distributed according to the terms of the GNU General Public License version 3.
+
+
+
+from ...util.model import BenchmarkModel
+from torchbenchmark.tasks import NLP
+import torch
+from .model import ModelArgs, Transformer
+import torch
+
+class Model(BenchmarkModel):
+    task = NLP.LANGUAGE_MODELING
+    
+    # TODO: Implement batching, right if I set as None I get NotImplementedError: Test eval is not implemented.
+    def __init__(self, test, device, jit=False, batch_size=32, extra_args=[]):
+        super().__init__(test=test, device=device, jit=jit, batch_size=32, extra_args=extra_args)
+        self.model_args = ModelArgs(vocab_size=32)
+        self.model = Transformer(self.model_args)
+
+
+        if device == "cuda":
+            torch.set_default_device("cuda")
+            self.model.to(torch.device("cuda"))
+        self.example_inputs = [(torch.tensor([[1, 1], [1,1]], dtype=torch.int), 1)]
+
+        
+    def get_module(self):
+        return self.model, *self.example_inputs
```

Comment:
Sorry, what is `*self.example_inputs` here? Does it return the very first element of the list?

Reply:
So I made this `*` change otherwise `test_llama_example_cuda` was broken but without it `test_llama_train/eval_cuda` were both working just fine. The intent is still to return all data but not sure how the `example` test is setup

Reply:
Fixed